docs/visibility.md

```diff
@@ -54,19 +54,12 @@ A List Filter contains <a class="tdlp" href="#search-attribute">Search Attribute
   - **BETWEEN ... AND**
   - **IN**
   - **ORDER BY**
```

Comment:
Server v1.20 will support Advanced Visibility capabilities on SQL databases; Will be updating this in a diff PR though; this is just FYI.

- ORDER BY is only supported on Elasticsearch and not on the SQL dbs. 
- Same with LIKE. It only works with Elasticsearch, and on Search attributes of type text IIRC. I think LIKE is still supported but is a major point of confusion with regards to usage. We can either remove it completely (which I see you have) or add clarifications etc. and only remove after support for it is dropped in a later release.

Reply:
I've already removed it, so I'll keep it as is. According to Alex, it doesn't work as one would hope.